modin/config/envvars.py

```diff
@@ -155,6 +155,26 @@ def _get_default(cls):
         return CpuCount.get()
 
 
+class EnablePartitionIPs(EnvironmentVariable, type=bool):
+    """
+    Whether to enable use of IPs for remote partitions of not
+    """
+
+    varname = "MODIN_ENABLE_PARTITIONS_IPS"
+
+    @classmethod
+    def _get_default(cls):
+        return False
```

Comment:
```suggestion
    default = False
```

`_get_default()` method is needed when default value is not a code-time constant (like CPU core count which differs based on machine code is run), for fully static defaults there is a special field